### ISSUE TITLE
Issue with importing builtins in a VS Code Jupyter notebook

### ISSUE THREAD

**Issue by danielgwilson** [uptrain-ai/uptrain#397]
## Describe the bug

Running into an issue with importing builtins in a VS Code Jupyter notebook.

Here's the error:

```python
from uptrain.framework.builtins import CheckResponseFacts

response_facts_check = CheckResponseFacts()
```

```sh
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
/Users/danielgwilson/local_git/patient-matching/python-notebooks/uptrain_sandbox.ipynb Cell 6 line 1
----> 1 from uptrain.framework.builtins import CheckResponseFacts
      3 response_facts_check = CheckResponseFacts()

File ~/local_git/patient-matching/.venv/lib/python3.11/site-packages/uptrain/framework/builtins.py:8
      6 from .checks import Check
      7 from uptrain.operators import Histogram
----> 8 from uptrain.operators import (
      9     ResponseFactualScore,
     10     ResponseCompleteness,
     11     ResponseRelevance,
     12     ContextRelevance,
     13     LanguageCritique,
     14     ToneCritique,
     15     ResponseCompletenessWrtContext,
     16     GuidelineAdherenceScore
     17 )
     19 # -----------------------------------------------------------
     20 # Context related
     21 # -----------------------------------------------------------
     23 CheckContextRelevance = lambda: Check(
     24     name="score_context_relevance",
     25     operators=[ContextRelevance()],
     26     plots=[Histogram(x="score_context_relevance")],
     27 )

ImportError: cannot import name 'GuidelineAdherenceScore' from 'uptrain.operators' (/Users/danielgwilson/local_git/patient-matching/.venv/lib/python3.11/site-packages/uptrain/operators/__init__.py)
```

Trying to follow the custom evals tutorial
https://docs.uptrain.ai/tutorials/customized-evaluations

## To Reproduce

1. Create a python notebook in VS Code and a new virtual environment
2. Follow the custom evals tutorial linked above
3. See error

## Expected behavior
No error

**Screenshots**
❌ If applicable, add screenshots to help explain your problem.

**Additional context**
N/A

**Comment by danielgwilson** [comment]
Update, seems to be missing from __index.pyi__

```python
__all__ = [
    # base
    "Operator",
    "ColumnOp",
    "TransformOp",
    "register_op",
    "register_custom_op",
    "get_output_col_name_at",
    "deserialize_operator",
    # drift
    "ParamsDDM",
    "ParamsADWIN",
    "ConceptDrift",
    # embs
    "Distribution",
    "UMAP",
    # clustering
    "Clustering",    
    # table
    "ColumnExpand",
    "ColumnComparison",
    # metrics
    "Accuracy",
    # similarity
    "CosineSimilarity",
    # vis
    "BarChart",
    "Histogram",
    "LineChart",
    "ScatterPlot",
    "MultiPlot",
    "Table",
    "CustomPlotlyChart",
    # language - also include all the subimports
    "language",
    "GrammarScore",
    "OpenaiEval",
    "PromptEval",
    "Embedding",
    "RougeScore",
    "DocsLinkVersion",
    "TextLength",
    "WordCount",
    "TextComparison",
    "KeywordDetector",
    "OpenAIGradeScore",
    "ModelGradeScore",
    "PromptGenerator",
    "TopicGenerator",
    "TextCompletion",
    "OutputParser",
    "ResponseFactualScore",
    "ContextRelevance",
    "ResponseRelevance",
    "ResponseCompleteness",
    "ResponseCompletenessWrtContext",
    "LanguageCritique",
    "ToneCritique",
    # io - also include all the subimports
    "io",
    "ExcelReader",
    "CsvReader",
    "JsonReader",
    "JsonWriter",
    "DeltaReader",
    "DeltaWriter",
    "BigQueryReader",
    "DuckDBReader",
    "BigQueryWriter",
    # code
    "code",
    "ParseCreateStatements",
    "ParseSQL",
    "ValidateTables",
    "ExecuteAndCompareSQL",
]

from .base import (
    Operator,
    ColumnOp,
    TransformOp,
    register_op,
    register_custom_op,
    get_output_col_name_at,
    deserialize_operator,
)
from .drift import ConceptDrift, ParamsADWIN, ParamsDDM
from .embs import Distribution, UMAP
from .clustering import Clustering
from .table import Table, ColumnExpand, ColumnComparison
from .metrics import Accuracy
from .similarity import CosineSimilarity
from .chart import (
    BarChart,
    LineChart,
    ScatterPlot,
    Histogram,
    MultiPlot,
    CustomPlotlyChart,
)

from . import language
from .language.grammar import GrammarScore
from .language.openai_evals import OpenaiEval, PromptEval
from .language.embedding import Embedding
from .language.rouge import RougeScore
from .language.text import (
    DocsLinkVersion,
    TextLength,
    TextComparison,
    KeywordDetector,
    WordCount,
)
from .language.model_grade import ModelGradeScore, OpenAIGradeScore
from .language.generation import PromptGenerator, TextCompletion, OutputParser, TopicGenerator
from .language.with_context import (
    ResponseFactualScore,
    ContextRelevance,
    ResponseCompleteness,
    ResponseRelevance,
    ResponseCompletenessWrtContext,
)
from .language.critique import LanguageCritique, ToneCritique

from . import io
from .io.base import CsvReader, JsonReader, DeltaReader, JsonWriter, DeltaWriter
from .io.excel import ExcelReader
from .io.bq import BigQueryReader, BigQueryWriter
from .io.duck import DuckDBReader

from . import code
from .code.sql import (
    ParseCreateStatements,
    ParseSQL,
    ValidateTables,
    ExecuteAndCompareSQL,
)
```

**Comment by sourabhagr** [comment]
Thanks @danielgwilson for flagging this! I just released a new version of UpTrain (v0.4.2) to fix the same. As you correctly mentioned, the issue was that the GuidelineAdherenceScore operator was not imported in the __ini__.pyi file. Please upgrade your version and check. Let me know if you run into any issues
